components/nimbus/ios/Nimbus/FeatureVariables.swift

```diff
@@ -3,6 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import Foundation
+import UIKit
```

Comment:
Just out of curiosity, I imagine something broke, it didn't compile and we needed to add this?

Reply:
Ah yes, thanks for the reminder. The code in this file is using some classes from `UIKit` (I think something about image-handling) but wasn't importing it explicitly. That was fine in our old iOS build setup because something else in the megazord was import it, but it broke when trying to compile this as its own package.